File: Assets/AltUnityTester/AltUnityDriver/Commands/InputActions/PressKey.cs
public class PressKey : AltBaseCommand
{
    Assets.AltUnityTester.AltUnityDriver.UnityStruct.KeyCode keyCode;
    float power;
    float duration;
    public PressKey(SocketSettings socketSettings, Assets.AltUnityTester.AltUnityDriver.UnityStruct.KeyCode keyCode, float power, float duration) : base(socketSettings)
    {
        this.keyCode = keyCode;
        this.power = power;
        this.duration = duration;
    }
    public void Execute(){
        Socket.Client.Send(toBytes(CreateCommand("pressKeyboardKey", keyCode.ToString(), power.ToString(), duration.ToString())));
        var data = Recvall();
        if (data.Equals("Ok"))
            return;
        HandleErrors(data);
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/InputActions/press_key.py
from altunityrunner.commands.base_command import BaseCommand
import time
class PressKey(BaseCommand):
    def __init__(self, socket,request_separator,request_end, keyName,power,duration):
        super().__init__(socket,request_separator,request_end)
        self.keyName=keyName
        self.power=power
        self.duration=duration
    
    def execute(self):
        print ('Press key: ' + self.keyName)
        data = self.send_data(self.create_command('pressKeyboardKey',self.keyName,self.power, self.duration ))
        return self.handle_errors(data)